.github/workflows/test-deploy.yml

```diff
@@ -22,10 +22,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.head.sha || github.ref }}
-      - uses: actions/setup-node@v2
-      - name: check lock
-        run: |
-          test $(cat package-lock.json | jq .lockfileVersion) == "2"
+      - run: jq --exit-status .lockfileVersion==2 < package-lock.json
```

Comment:
```suggestion
      - name: check that npm lock file is on version 2
        run: jq --exit-status .lockfileVersion==2 < package-lock.json
```

```js
if (check == true) { // if the check variable is true
  return;
}
```

Reply:
I take it back the action's job name is sufficient for my `at a glance` determination of possible reasons for a check failure which is why names are nice

Reply:
I used to write all my steps with names until I joined Iterative. 😅 In sentence case, for consistency with the default step names.